components/services/bat_ledger/public/interfaces/bat_ledger.mojom

```diff
@@ -100,6 +100,8 @@ interface BatLedger {
 
   SetCatalogIssuers(string info);
   AdSustained(string info);
+
```

Comment:
nit: why the extra space here?

Reply:
It looked like vertical whitespace was being used in that file to group similar functions, but I'll remove it.